topics/admin/tutorials/reference-genomes/tutorial.md

```diff
@@ -0,0 +1,667 @@
+>        items:
+>          - dbkey: sacCer3
+>            name: 'S. cerevisiae Apr. 2011 (SacCer_Apr2011/sacCer3)'
+>        data_table_reload:
+>          - all_fasta
+>          - __dbkeys__
+>    ```
+>    {% endraw %}
+>
+>    > <details-title>What do these config file compontents mean?</details-title>
+>    >
+>    > The `run-data-managers` config file options correspond to the options in the data manager tool XML file. To locate the tool XML file for the Fetch Genomes data manager, you can search for it in [the Tool Shed](https://toolshed.g2.bx.psu.edu) the same way you did when installing it via the UI. You can also open the data manager's tool form in the UI as if to run it, and, using the drop-down menu at the top right of the tool form, click "See in Tool Shed". From the Tool Shed, you can click the **Development repository** link and browse to the [tool XML file, `data_manager/data_manager_fetch_genome_all_fasta_dbkeys.xml`](https://github.com/galaxyproject/tools-iuc/blob/8487d2c73793be0afa5b34388b122e686ac8a094/data_managers/data_manager_fetch_genome_dbkeys_all_fasta/data_manager/data_manager_fetch_genome_all_fasta_dbkeys.xml).
+>    >
+>    > `run-data-managers` config file component          | Purpose
+>    > ---                                                | ---
+>    > `id`                                               | Data manager full (shed) tool ID, this can be found in `shed_data_manager_conf.xml`
+>    > `params`                                           | Data manager tool params, these correspond to `<param>` tags in the tool XML file. Nested paramaters are specified using a pipe character (`|`).
+>    > param `dbkey_source|dbkey`                         | Value of `<param name="dbkey" ...>` in `<conditional name="dbkey_source">`.
+>    > param `sequence_name`                              | Value of `<param name="sequence_name" ...>`.
+>    > param `reference_source|reference_source_selector` | Value of `<param name="reference_source_selector" ...>` in `<conditional name="reference_source">`.
+>    > param `reference_source|requested_dbkey`           | Value of `<param name="requested_dbkey" ...>` in `<conditional name="reference_source">`.
+>    > `items`                                            | A list of variables to template in to `params`, referenced in param fields with {% raw %}`{{ item }}`{% endraw %}. In the case of genomes, for example, you can run this DM with multiple genomes, or you could give multiple URLs.
+>    > `data_table_reload`                                | Names of the data tables you want to reload after your DMs are finished running. This can be important for subsequent data managers.
+>    >
+>    {: .details}
+>
+>
+> 2. Run the Genome Fetch DM with `run-data-managers`:
+>
+>    > <code-in-title>input: bash</code-in-title>
+>    > ```console
+>    > run-data-managers -g https://your-galaxy -a <api-key> --config fetch-sacCer3.yml
```

Comment:
I approve! But FWIW I did this because it's used in a bunch of other tutorials. I can make a separate PR updating those.

But also I prefer `example.org` over `example.com`, any objection?

Reply:
>  I can make a separate PR updating those.

just toss it in this one it's fiiiineee

> any objection?

none